packages/plugins/AddressSearch/src/components/Results.vue

```diff
@@ -117,6 +154,53 @@ export default Vue.extend({
       return Boolean(searchService?.queryParameters?.maxFeatures)
     },
     emTitleByInput,
+    getNextElementId(
+      index: number,
+      nextInnerDex: number,
+      featureListLength: number
+    ): string {
+      if (nextInnerDex === -1 && index === 0) {
+        return 'polar-plugin-address-search-input'
+      }
+
+      let nextIndex: number
+      let usedInnerDex: number
+      if (nextInnerDex === -1) {
+        nextIndex = index - 1
+        usedInnerDex = this.featureListsWithCategory[nextIndex].length - 1
+      } else if (nextInnerDex === featureListLength) {
+        nextIndex =
+          index + 1 === this.featureListsWithCategory.length ? 0 : index + 1
+        usedInnerDex = 0
+      } else {
+        nextIndex = index
+        usedInnerDex = nextInnerDex
+      }
```

Comment:
This doesn't always work in the DISH client. Disable CORS in your browser to test the client appropriately.

Test cases:

- Enter "Rathaus" into the search. Only the first search result category is reachable by the arrow keys.
- Enter "Rathausm" into the search. The second category can be reached, but there is no way back up with the arrow keys.

In both scenarios, the open/close buttons for further results are not reachable by the arrow keys, and neither are the results reachable from the open/close buttons with arrow keys. Especially the latter case seems unintuitive since we're still "in" the thing we're navigating.

Reply:
The problems regarding the results occur because the full list is being rendered (the list items are in the DOM) meaning that the open / close buttons don't properly hide them.  
If all the results are being shown for the full list, one can navigate between the two lists.

However, I fully understand that the behaviour is weird regarding the open / close buttons so I'll take a look into them being also only navigable via arrow keys.

Reply:
This should now work as intended in both Meldemichel and DISH.

If you have any suggestions to clean up the way the id of the next element is retrieved, I am open to 🏌️⛳

Reply:
- There a small issue left in DISH where arrow navigation doesn't work for singular result in later categories. E.g. search for "2-20/Rathausstraße". The arrow key won't properly work in this situation.
- On pressing space on a selected list item, the list item shows an effect, but nothing happens. It was like that before, but would you include a quick fix while we're here?
- "If you have any suggestions to clean up the way the id of the next element is retrieved, I am open to 🏌️⛳" – I assume it would be shorter to write functions `getNextElement` and `getPreviousElement` that get the current element's ID and just navigate the DOM with `.nextSibling` and `.previousSibling`. Since we have one level of nesting (`<ul>`) to consider, we'd also need `parentElement` and `children` for switching lists and some skipping on at least `div` and `hr`. 🤔

Reply:
- I'll take a look into that
- If it is a rather easy fix, sure. Otherwise I will write an issue
- That doesn't really seem easier tbh :D We can ☎️ later

Reply:
- Confirmed.
- It still ripples on my machine.
- Created. https://github.com/Dataport/polar/issues/26

Reply:
The ripple should be dealt with in all situations fbfb8eba